tests/view_tests/tests/test_debug.py

```diff
@@ -825,6 +825,48 @@ def test_template_encoding(self):
             reporter.get_traceback_text()
             m.assert_called_once_with(encoding='utf-8')
 
+    def test_innermost_exception_traceback_none(self):
+        try:
+            source = (
+                "def funcName():\n  try:\n    raise RuntimeError('my error')\n"
+                "  except Exception as exc:\n    new_exc = RuntimeError('my context')\n"
+                "    exc.__context__ = new_exc\n    raise\nfuncName()"
+            )
+            namespace = {}
+            code = compile(source, 'generated', 'exec')
+            exec(code, namespace)
+        except Exception:
+            exc_type, exc_value, tb = sys.exc_info()
+        request = self.rf.get('/test_view/')
+        reporter = ExceptionReporter(request, exc_type, exc_value, tb)
+        frames = reporter.get_traceback_frames()
+        last_frame = frames[-1]
+        self.assertEqual(last_frame['context_line'], '<source code not available>')
+        self.assertEqual(last_frame['filename'], 'generated')
+        self.assertEqual(last_frame['function'], 'funcName')
+        self.assertEqual(last_frame['lineno'], 3)
```

Comment:
We didn't change anything in building frames so it's enough to check that frames are not empty.